actix-web-opentelemetry/Cargo.toml

```diff
@@ -0,0 +1,70 @@
+[package]
+name = "actix-web-opentelemetry"
+version = "0.21.0"
+description = "OpenTelemetry integration for Actix Web apps"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/actix-web-opentelemetry"
+repository = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/actix-web-opentelemetry"
+readme = "README.md"
+categories = ["api-bindings"]
+keywords = ["actix", "actix-web", "opentelemetry", "jaeger", "prometheus"]
+license = "Apache-2.0"
+edition = "2021"
+rust-version = "1.75.0"
+
+[features]
+metrics = ["opentelemetry/metrics"]
+metrics-prometheus = [
+  "metrics",
+  "opentelemetry-prometheus",
+  "prometheus",
+  "dep:opentelemetry_sdk",
+  "dep:tracing",
+]
+sync-middleware = []
+
+[dependencies]
+actix-http = { version = "3.0", default-features = false, features = [
+  "compress-zstd",
+] }
+actix-web = { version = "4.0", default-features = false, features = [
+  "compress-zstd",
+] }
+awc = { version = "3.0", optional = true, default-features = false, features = [
+  "compress-zstd",
+] }
+futures-util = { version = "0.3", default-features = false, features = [
+  "alloc",
+] }
+opentelemetry = { version = "0.28", default-features = false, features = [
+  "trace",
+] }
+opentelemetry-prometheus = { version = "0.28", optional = true }
+opentelemetry-semantic-conventions = { version = "0.28", features = [
+  "semconv_experimental",
+] }
+opentelemetry_sdk = { version = "0.28", optional = true, features = [
```

Comment:
@lalitb raised an important point about this sdk dependency in the community call. Instrumentations should only rely on the opentelemetry crate.
However, lets do the lift-and-shift now, and then address issues in subsequent releases.